config/src/client.rs

```diff
@@ -107,6 +107,10 @@ impl Default for ConfigurationProxy {
     }
 }
 
+// TODO: explain why these values were chosen.
```

Comment:
This was rather arbitrary tbh, just *unreasonably* small values I've drawn from my head. Maybe it invites further discussion.